tensorflow/examples/learn/wide_n_deep_tutorial.py

```diff
@@ -151,12 +152,11 @@ def input_fn(df):
   continuous_cols = {k: tf.constant(df[k].values) for k in CONTINUOUS_COLUMNS}
   # Creates a dictionary mapping from each categorical feature column name (k)
   # to the values of that column stored in a tf.SparseTensor.
-  categorical_cols = {
-      k: tf.SparseTensor(
-          indices=[[i, 0] for i in range(df[k].size)],
-          values=df[k].values,
-          dense_shape=[df[k].size, 1])
-      for k in CATEGORICAL_COLUMNS}
+  categorical_cols = {k: tf.SparseTensor(
+      indices=[[i, 0] for i in range(df[k].size)],
+      values=df[k].values,
+      shape=[df[k].size, 1])
```

Comment:
The SparseTensor API has recently changed. `dense_shape` is the new, correct keyword name to use. See: https://www.tensorflow.org/versions/master/api_docs/python/sparse_ops/sparse_tensor_representation#SparseTensor
I am going to close this PR. Please let me know if you have objections